src/test/cls_2pc_queue/test_cls_2pc_queue.cc

```diff
@@ -709,15 +709,20 @@ TEST_F(TestCls2PCQueue, MultiProducer)
           librados::ObjectWriteOperation op;
           const auto max_elements = 42;
           const std::string marker;
-          bool truncated = false;
+          bool truncated = true;
```

Comment:
so the real bugfix is here (ensure queue_list_entries will be attempted at least once)?

Reply:
yes. and same fix at line 769